packages/zoe/src/typeGuards.js

```diff
@@ -347,7 +347,7 @@ export const ZoeServiceI = M.interface('ZoeService', {
   getInstallationForInstance: M.callWhen(M.await(InstanceHandleShape)).returns(
     M.eref(M.remotable('Installation')),
   ),
-  getBundleIDFromInstallation: M.call(InstallationShape).returns(
+  getBundleIDFromInstallation: M.callWhen(M.await(InstallationShape)).returns(
```

Comment:
ISTR @dtribble pointing out a security concern where a client can pass a promise and not resolve it, tying up resources in zoe.

Much of the zoe API already accepts promises in this way, so perhaps we have already accepted the risk. But I'd like to be sure.

@warner ? @erights ? thoughts?